screens/PointsHistoryScreen.js

```diff
@@ -0,0 +1,44 @@
+import * as Font from 'expo-font';
+import React from 'react';
+import { AsyncStorage, ScrollView, Text, View } from 'react-native';
+import getCustomerTransactions from './historyHelpers';
+
+import Transactions from '../components/Transactions';
+
+export default class PointsHistoryScreen extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      transactions: []
+    };
+  }
+
+  async componentDidMount() {
+    const userId = await AsyncStorage.getItem('userId');
+    getCustomerTransactions(userId).then(transactions => {
+      this.setState({ transactions });
+    });
+
+    Font.loadAsync({
+      Poppins: require('../assets/fonts/Poppins-Regular.ttf')
+    });
+  }
+
+  render() {
+    return (
+      <View>
+        <Text> RECENT TRANSACTIONS</Text>
```

Comment:
Super nitpick but this should go inside ScrollView, esp. after you split out transactions into the `recents` state object